lib/debug/observatory_debugger.dart

```diff
@@ -472,6 +472,9 @@ class ObservatoryIsolate extends DebugIsolate {
 
   String get name => isolateRef.name;
 
+  String get displayName =>
+    name.contains(r'$') ? "${name.replaceAll(r'$', ':')}()" : name;
```

Comment:
Will do. Isolate names are something like `foo_app.dart$main`. This cleans them up slightly to `foo_app.dart:main()`; I'll add a note.